### ISSUE TITLE
Improve text accessibility for top text and note style

### ISSUE THREAD

**Issue by gaby-dominguez** [gaby-dominguez/figma-feedback#862]
**Subject:** UX / Design
**Priority:** medium
**Author:** Morgan  Rowney
**Date:** 2026-04-21T15:11:18.184Z

**Insight:** The top text lacks accessibility; a style change is suggested.

**Sprint Title:** Enhance accessibility and styling for top text

**Original Comment:**
> @Jacob Sørensen This top text isn't accessible. If you take a look at my designs on this the grey is accessible there.
> 
> Also wonder if we switch the note style from headline to large body (again I have this over on my side)

**Component screenshot:**
![component](https://figma-alpha-api.s3.us-west-2.amazonaws.com/images/9f7073bc-5459-47a0-bd49-a69555268053)

**Component link:** https://www.figma.com/design/ffl2nYDWOrgYHjswogKXTl?node-id=4244%3A12457

---
*Source: Figma comment `1727874209`*
